File: resourcespoet/src/test/java/com/commit451/resourcespoet/AttrTest.java
package com.commit451.resourcespoet;

import org.junit.Test;

import java.util.ArrayList;

/**
 * Tests the resources creation
 */
public class AttrTest {

    @Test
    public void attrTest() throws Exception {
        ArrayList<Attr.Format> formats = new ArrayList<>();
        formats.add(Attr.Format.STRING);
        formats.add(Attr.Format.REFERENCE);
        Attr attr = new Attr("font", formats);
        ResourcesPoet poet = ResourcesPoet.create()
                .addAttr(attr);

        TestUtil.assertEquals("attr.xml", poet);
    }
}

File: src/test/kotlin/com/commit451/resourcespoet/AttrTest.kt
package com.commit451.resourcespoet

import org.junit.Test
import java.util.*

/**
 * Tests the resources creation
 */
class AttrTest {

    @Test
    fun attrTest() {
        val formats = ArrayList<Attr.Format>()
        formats.add(Attr.Format.STRING)
        formats.add(Attr.Format.REFERENCE)
        val attr = Attr("font", formats)
        val poet = ResourcesPoet.create()
                .addAttr(attr)

        TestUtil.assertEquals("attr.xml", poet)
    }
}
